docs/setup/self-hosted/docker-desktop.mdx

```diff
@@ -21,6 +21,12 @@ Follow the steps to set up MindsDB in Docker Desktop.
 
 ### Install the MindsDB Docker Desktop Extension
 
+<Tip>
+If you are a Windows user, ensure that you have enabled Developer Mode under settings before installing the extension.
+
+<img src="/assets/docker/docker_desktop/enable-win-dev-mode.png"/>
```

Comment:
Let's add an image with a screenshot where Developer Mode is enabled, like below:

![image](https://github.com/mindsdb/mindsdb/assets/109554435/ee0c6217-5e22-4c9d-98bc-2c21858ce83e)

Reply:
@martyna-mindsdb Done.